actors/runtime/runtime.go

```diff
@@ -33,8 +33,8 @@ type Runtime interface {
 	// Look up the code ID at an actor address.
 	GetActorCodeCID(addr addr.Address) (ret cid.Cid, ok bool)
 
-	// Randomness returns a (pseudo)random string for the given epoch and tag.
-	GetRandomness(epoch abi.ChainEpoch) abi.RandomnessSeed
+	// Randomness returns a (pseudo)random byte array from the chain at a given epoch
+	DrawRandomnessFromChain(epoch abi.ChainEpoch) []byte
```

Comment:
I am confused by the removal of the `RandomnessSeed` type but still having a `Randomness` type, and the `DrawRandomness` method not returning a value of that type. Also, reference to the blockchain is intentionally minimised inside the VM (everything inside should roughly make sense if there was some other consensus distributed database mechanism).

So how about `DrawRandomSeed(epoch) abi.RandomSeed` (unless I've misinterpreted)?

Reply:
Yeah I think the `randomnessSeed` vs `Randomness` distinction is confusing so I'm going to move to using Randomness everywhere.